Integrate grunt tasks

Vercryger opened the pull request:
I did some improvements as needed on #4. Also, I added Grunt tasks to make builds for both, development and production.

Vercryger committed:
Integrate grunt tasks

bluzi commented:
hi @Vercryger, thank you for your contribution! 
could you please elaborate on what you did with grunt and what does it add to the project?
i don't really understand why we need browserify/babel, as this is a chrome extension, and we can use the newest JS standard available for chrome.

also, please elaborate on the test you've made to ensure your code works.

thanks!

Vercryger committed:
Merge remote-tracking branch 'bluzi/master' into code-readability

Vercryger commented:
Hey @bluzi , you're welcome!
I know this is just a chrome extension, but the source still JavaScript, and there are a lot of ways you  can write the code. I've just added Grunt to integrate ES5 standards, like `import`, as you may noticed, I removed your *js* folder, because you were including a third-party library, this is not the best way to do it, that's why I added it as a new NPM dependency. In this case, your source files should be ONLY the `inject.js`.

By the other hand, grunt is a great option to make distribution builds. I've tested this just to adding the distribution folder as a new chrome extension.
